### PR TITLE
Corrected incorrect translations and added some

### PR DESCRIPTION
**Pull request by Tasm-Devil**
Should the error messages from files in the py folder also be translated?

### CODE REVIEW

File: locale/de_DE.po [hunk 1]
```diff
@@ -677,144 +672,132 @@ msgstr "Das Interval ist nicht im Bereich 0.0020 bis 10.24"
...
 
 #: ports/nrf/common-hal/bleio/Device.c:262
-#, fuzzy
 msgid "Failed to discover services"
 msgstr "Es konnten keine Dienste gefunden werden"
 
-#: ports/nrf/common-hal/bleio/Device.c:268
-#: ports/nrf/common-hal/bleio/Device.c:302
-#, fuzzy
+#: ports/nrf/common-hal/bleio/Device.c:267
```
Comment (Tasm-Devil) [inline locale/de_DE.po]: and here. and so on.
Where does this come from? I took the latest version.

File: locale/de_DE.po [hunk 2]
```diff
@@ -335,15 +335,15 @@ msgstr "Baudrate wird nicht unterstützt"
 
 #: ports/atmel-samd/common-hal/busio/UART.c:67
 msgid "bytes > 8 bits not supported"
-msgstr "bytes mit merh als 8 bits werden nicht unterstützt"
+msgstr "bytes mit mehr als 8 bits werden nicht unterstützt"
 
 #: ports/atmel-samd/common-hal/busio/UART.c:73
-#: ports/nrf/common-hal/busio/UART.c:91
+#: ports/nrf/common-hal/busio/UART.c:118
```
[the remark this answers is not in the document]
Comment (tannewt) [reply locale/de_DE.po]: The line numbers are automatically updated when someone runs `make translate` on the repo. You don't need to update them yourself.

[the remark this answers is not in the document]
Comment (Tasm-Devil) [reply locale/de_DE.po]: And what does this means to the wrong linenumbers, if you commit/accept my changes?
Will the translation still work?

[the remark this answers is not in the document]
Comment (tannewt) [reply locale/de_DE.po]: Yes they will. The translation mapping is based on `msgid` and will be ignored if the translation is marked fuzzy.

**Pull request merged**